js/test/hooks/useCertifications.test.ts

```diff
@@ -1,16 +1,25 @@
 import { fetch } from "../../browser";
 import { useCertifications } from "../../hooks/useCertifications";
+import { filterRelevantForOperators } from "../../models/certification";
 import {
   certificationDataFactory,
   certificationFactory,
 } from "../helpers/factory";
 import { PromiseWithResolvers } from "../helpers/promiseWithResolvers";
 import { renderHook, waitFor } from "@testing-library/react";
+import { DateTime } from "luxon";
 
 jest.mock("../../browser", () => ({
   fetch: jest.fn(),
 }));
 
+// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+jest.mock("../../models/certification", () => ({
+  ...jest.requireActual("../../models/certification"),
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+  filterRelevantForOperators: jest.fn((cs, _line) => cs),
```

Comment:
This is a pure function, so you shouldn't need to mock it or assert that it was called.

You could instead use test data that includes a BL rail cert and a OL rail cert, and assert that the hook only returns the BL cert. (This would probably be easier to do in the existing test case rather than writing a second case.)

Reply:
6137986

I was doing the mock originally to prevent the helper from getting called even though it's pure (so if its behavior changed the test would still pass), but I admit that isn't strictly necessary. Changed!